src/meta/ActiveHostsMan.cpp

```diff
@@ -258,36 +249,5 @@ ErrorOr<nebula::cpp2::ErrorCode, HostInfo> ActiveHostsMan::getHostInfo(kvstore::
   return HostInfo::decode(hostValue);
 }
 
-nebula::cpp2::ErrorCode LastUpdateTimeMan::update(kvstore::KVStore* kv,
```

Comment:
We'd better save this interface, just modify the parameter type. It would be implicit that we write a `lastUpdateTimeKey`, the `LastUpdateTimeMan::update` is more explicit.

Reply:
I intend to eliminate the problem of writing many times.If a function only retains the function of assembling key values, should it really exist?